### PR TITLE
Identity (Selenium tests)

### PR DESCRIPTION
**Pull request by fatmaBouchekoua**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | This PR adds the scenario (identity.js) that exists in the repo test/Selenium/specs
| Type?         | new feature
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | /
| How to test?  | npm run selenium-test -- --URL=URLFrontOffice

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9341)

### CODE REVIEW

File: tests/E2E/prepare-shop.php [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+define('_PS_MODE_DEV_', false);
```
Comment (Quetzacoalt91) [inline tests/E2E/prepare-shop.php:27]: This file already exist in `tests/E2E`, why don't you reuse it?

Comment (fatmaBouchekoua) [reply tests/E2E/prepare-shop.php:27]: You mean tests/Selenium?

Comment (Quetzacoalt91) [reply tests/E2E/prepare-shop.php:27]: Yes, you're right.

Comment (fatmaBouchekoua) [reply tests/E2E/prepare-shop.php:27]: This repository (tests/Selenium) will be removed soon after migrating all tests to the repository E2E, that's why I didn't use it.

**Comment by fatmaBouchekoua** [comment]
Scenario reviewed by @hadrich-hatem

**Comment by fatmaBouchekoua** [comment]
@mbadrani this PR is ready to be merged

**Review by mbadrani — APPROVED** [review]
already reviewed

**Pull request merged**